build.gradle

```diff
@@ -29,6 +29,17 @@ repositories {
     maven { url "https://ladysnake.jfrog.io/artifactory/mods" }
     maven { url = "https://dvs1.progwml6.com/files/maven/" }
     maven { url = "https://modmaven.dev" }
+    exclusiveContent {
```

Comment:
Happy to revert this change, though it was really nice to not need to manually download BCLib.

Reply:
Does this pull the correct files? I do not upload dev-versions of BCLib to modrinth...